### DISCUSSION TITLE
Premade characters_npcbot table

### DISCUSSION THREAD

**Discussion by ZeroCelsius** [trickerer/Trinity-Bots#220] [Ideas]
Love your work, it's excellent. Thank you very much for providing this labor 🙇‍♂️

Do you by chance have (or have any interest in) a premade characters_npcbot table, for those that want to quickly populate their servers?

**Comment by trickerer** [comment]
Not really. I mean it's not enough to just include `characters_npcbot` table in distribution. Bots are creatures spawned in the world so they occupy entries in `creature` table also. Plus, they can't have any templated equipment; they rely on `item_instance` table for items they use.

A 'quick start' sql script is possible to create... I think. Spawning creatures, creating items and stuff. The most time would be consumed by looking for item entries for bots to use and selecting spawn locations anyway.

**Reply by ZeroCelsius** [reply]
I thought about that last part too, when I was trying to think up "lore appropriate" spawn locations. I eventually just went with leaving my bots on GM Island and spawning Lagretta outside the capital taverns for that "mercenaries looking for work in a tavern " feel, almost like a Mercenaries Guild representative of sorts.

Thanks again!
